src/_pytest/reports.py

```diff
@@ -251,6 +257,50 @@ def _report_unserialization_failure(
     raise RuntimeError(stream.getvalue())
 
 
+def _format_failed_longrepr(
+    item: Item, call: CallInfo[None], excinfo: ExceptionInfo[BaseException]
+):
+    if call.when == "call":
+        longrepr = item.repr_failure(excinfo)
+    else:  # exception in setup or teardown
+        longrepr = item._repr_failure_py(
+            excinfo, style=item.config.getoption("tbstyle", "auto")
+        )
+    return longrepr
+
+
+def _format_exception_group_all_skipped_longrepr(
+    item: Item,
+    excinfo: ExceptionInfo[BaseException],
+    exceptions: list[Skipped],
```

Comment:
Since `exceptions` is not modified by this function, prefer `Sequence[Skipped]` over `list[Skipped]`.